### DISCUSSION TITLE
Confusing man page

### DISCUSSION THREAD

**Discussion by bound-variable** [Morganamilo/paru#717] [General]
The man page says,

> If no arguments are provided 'paru -Syu' will be performed.

But this isn't the entire story. Running `paru <target>` will launch an interactive menu, whereas running `paru -Syu <target>` will not. So `paru` and `paru -Syu` are not equivalent.

I was confused by this for awhile. My opinion is that the documentation can be made clearer.

**Comment by Morganamilo** [comment]
<target> is an argument.

**Comment by bound-variable** [comment]
You're right. I didn't think about that. However, in the README it says,

> paru <target> -- Interactively search and install <target>.

> paru -- Alias for paru -Syu.

This is confusing. If paru runs paru -Syu, then why the different behavior? If I run paru -Syu <target> the behavior is different than from paru <target>

**Comment by Morganamilo** [comment]
The readme says `paru <target> -- Interactively search and install <target>.` Not sure where you got what you're seeing.

**Reply by bound-variable** [reply]
> The readme says `paru <target> -- Interactively search and install <target>.` Not sure where you got what you're seeing.

https://github.com/Morganamilo/paru#examples

**Reply by Morganamilo** [reply]
Again though what you said is not what the readme says.

**Reply by bound-variable** [reply]
Indeed. I copied and pasted. Not sure what happened there.
